.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     'prettier',
     'prettier/react',
     'prettier/@typescript-eslint',
+    'plugin:svg-needs-viewBox/all',
```

Comment:
suggestion: I kind of think this should be called `plugin:paste` so it can contain all our custom rules in the future.

Reply:
I was considering that as well, and I'm into it, especially if we think we'll have more of these. And the repetition was driving me nuts anyway 🤣 
:paste:

Reply:
I would like it to be more specific tho. how about `plugin:paste-eslint`?

Reply:
from standup: since we'll probably be publishing an external npm package of this plugin, we'll be naming this `paste-internal`